drivers/input/touchscreen/goodix.h

```diff
@@ -78,6 +78,7 @@ struct goodix_chip_data {
 struct goodix_ts_data {
 	struct i2c_client *client;
 	struct input_dev *input_dev;
+	u32 num_keys;
```

Comment:
Do you envision this value being used outside the configure function? As it stands, it could just be a local.